app/src/main/java/backgammon/DoublingCube.java

```diff
@@ -0,0 +1,26 @@
+package backgammon;
+
+public class DoublingCube {
+
+  private int cubeValue;
+
+  public DoublingCube(){
+    cubeValue = 1;
+  }
+
+
+  public int getCubeValue(){
+    return cubeValue;
+  }
+
+  public void setCubeValue(int x){
+    this.cubeValue = x;
+  }
+
+  public void doubleCubeValue(){
```

Comment:
Rename this to `doubleStakes` instead and add the check here that caps the doubling cube at a max of 64.